code/mobility-services/mobility-services.js

```diff
@@ -14,63 +28,62 @@ $(document).on('knack-view-render.any', function (event, view, data) {
       this.value = this.value.replace(/[^0-9\s-]+/g, '');
     }
     if (this.value.charAt(3) != " "){
-      this.value = this.value.replace(/^(.{3})(.*)$/, "$1 $2"); // replace if no space at 4th
+      this.value = this.value.replace(/^(.{3})(.*)$/, "$1 $2"); // replace if no space at 4th position
     }
-    if (this.value.charAt(6) != " "){ // replace if no spaces at 7th
+    if (this.value.charAt(6) != " "){ // replace if no spaces at 7th position
       this.value = this.value.replace(/^(.{6})(.*)$/, "$1 $2");
     }
 
   });
   // Validation
   $("input#field_33").attr('maxlength', 11);
   $("input#field_33").attr('placeholder',"___ __ ____");
+
 });
 
 /********************************************************/
-/***** Show character limit on paragraph text type ******/
+/********** Show character limit on text type ***********/
 /********************************************************/
 function showCharacterLimit(view_id, field_id, charLimit) {
-  $( document ).ready(function() {
-    const charLeftText = "<p class='typed-chars'>" + charLimit +  " characters allowed</p>";
-    $(".kn-form.kn-view." + view_id + " form #" + field_id).after(charLeftText);
-    $(".kn-form.kn-view."+ view_id + " form #" + field_id).on('input',function(e){
-      const $input = $(this);
-      const charCount = $input.val().length;
-      const totalChar = charLimit - charCount;
-      if (charCount > charLimit) {
-        $input.siblings('.typed-chars').css({"color": "#ff0000", "font-weight": "bold"});
-        $input.siblings('.typed-chars').text(Math.abs(totalChar) + " characters over limit");
-      } else {
-        $input.siblings('.typed-chars').css({"color": "#4a4a4a", "font-weight": "normal"});
-        if (charCount == 0) {
-          $input.siblings('.typed-chars').text(totalChar + " characters allowed");
-        } else {
-          $input.siblings('.typed-chars').text(totalChar + " characters left");
-        }
-      }
+  // Function returns list as message string and css dictionary based on the input length and character limit
```

Comment:
Please use proper comment formatting for this function section

Reply:
Huddled on it that - new line is `/* */` is preferred